CDP4PluginPackager/CDP4PluginPackager.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
please add more tags related to
  - authors
  - company
  - copyright
  - license

etc.. see CDP4Services projects to get inspiration